k8s_itests/cmds.py

```diff
@@ -0,0 +1,12 @@
+import subprocess
```

Comment:
don't know about others, but it really irks me that our `*_itests` folders all precede the main source folder `paasta_tools`, this makes bunch of things less efficient, like navigating search results, how about we rename all of it to `tests_integration_*`? cc @solarkennedy @Rob-Johnson @vkhromov @chlgit